test/Microsoft.AspNet.Mvc.Razor.Host.Test/TestFiles/Input/MultipleModels.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@model ThisShouldNotBeGenerated
```

Comment:
Huuuu? This is an impossible case due to the parser barfing on multiple models isn't it??